op-guide/gc.md

```diff
@@ -0,0 +1,84 @@
+---
+title: TiDB Garbage Collection (GC)
+category: advanced
+---
+
+# TiDB Garbage Collection (GC)
+
+TiDB uses MVCC to control concurrency. When you update or delete data, the data is not deleted immediately but is saved for a period during which it can be read. Thus the write operation and the read operation are not mutually exclusive and it is possible to read the previous data.
+
+The data versions whose duration exceeds a specific time and that are not used any more will be cleared, otherwise they will occupy the disk space, affecting the system performance. TiDB uses Garbage Collection (GC) to clear the obsolete data. 
+
+## Working mechanism
+
+GC runs periodically on TiDB. When a TiDB server is started, a `gc_worker` is enabled in the background. In each TiDB cluster, one `gc_worker` is elected to be the leader which is used to maintain the GC status and send GC commands to all the TiKV Region leaders. 
```

Comment:
A gc worker is started in the  background, not enabled. These text are moved from history-read.md, around line 34.